### PR TITLE
Added apache redirects requested in #3484

### PR DESCRIPTION
**Pull request by cwdavies**
Add redirects for broken links, that includes policy-compliance pages and the old Ask CFPB search URL

## Additions

Created permanent apache redirects for the following:
/about-us/advisory-groups/ -> /about-us/advisory-committees/
/askcfpb/search/ -> /ask-cfpb/search/
/complaints/ -> /data-research/consumer-complaints/
/foia/quarterly/ -> /foia-requests/foia-electronic-reading-room/cfpb-foia-quarterly-reports/
/how-to-prepare-your-finances-for-buying-a-home/ -> /owning-a-home/process/prepare/
/owning-a-home/loan-options/FHA-loans/ -> /owning-a-home/loan-options/fha-loans/
/practitioner-resources/students/repay/ -> /paying-for-college/repay-student-debt/
/supervision/ -> /jobmanager/supervision.html
/wp-content/uploads/ - > https://s3.amazonaws.com/files.consumerfinance.gov/f

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: cfgov/apache/conf.d/redirects.conf [hunk 1]
```diff
@@ -982,19 +930,18 @@ RedirectMatch permanent (?i)^/es/guia-asesor-financiero https://pueblo.gpo.gov/C
...
 RedirectMatch permanent (?i)^/managing-someone-elses-money([/]?)$ /consumer-tools/managing-someone-elses-money/
 
-
 # Redirect old Tax Preparers URL to its new Wagtail home
 RedirectMatch permanent (?i)^/tax-preparer-resources([/]?)$ /practitioner-resources/resources-for-tax-preparers/
 
 # Redirect FY13-17 Strategic Plan page to PDF to move out of WordPress
 # See discussion here: https://GHE/CFGOV/platform/issues/710
-Redirect /strategic-plan/ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
+RedirectMatch permanent (?)^/strategic-plan/?$ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
```
Comment (higs4281) [inline cfgov/apache/conf.d/redirects.conf]: did you intend `(?i)` here?

Comment (higs4281) [reply cfgov/apache/conf.d/redirects.conf]: Also, not sure we need the trailing `?` in the match string, but the `$` makes sense to ignore downstream.

File: cfgov/apache/conf.d/redirects.conf [hunk 2]
```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
-
 # Common misspellings
 RedirectMatch (?i)(.*)morgage(.*) $1mortgage$2
 
 Redirect /assets https://s3.amazonaws.com/files.consumerfinance.gov/a/assets
+
+# Redirects, requested by Rachael Keeler in July 2019
+# https://GHE/CFGOV/platform/issues/3484
+RedirectMatch permanent (?i)^/about-us/advisory-groups/(.*) /about-us/advisory-committees/$1
+Redirec permanent /about-us/blog/understand-your-options-tips-student-loan-borrowers-disabilities /about-us/blog/tips-student-loan-borrowers/
```
Comment (higs4281) [inline cfgov/apache/conf.d/redirects.conf]:
```suggestion
Redirect permanent /about-us/blog/understand-your-options-tips-student-loan-borrowers-disabilities /about-us/blog/tips-student-loan-borrowers/
```

Comment (cwdavies) [reply cfgov/apache/conf.d/redirects.conf]: Good catch

File: cfgov/apache/conf.d/redirects.conf [hunk 3]
```diff
@@ -982,19 +930,18 @@ RedirectMatch permanent (?i)^/es/guia-asesor-financiero https://pueblo.gpo.gov/C
...
 RedirectMatch permanent (?i)^/managing-someone-elses-money([/]?)$ /consumer-tools/managing-someone-elses-money/
 
-
 # Redirect old Tax Preparers URL to its new Wagtail home
 RedirectMatch permanent (?i)^/tax-preparer-resources([/]?)$ /practitioner-resources/resources-for-tax-preparers/
 
 # Redirect FY13-17 Strategic Plan page to PDF to move out of WordPress
 # See discussion here: https://GHE/CFGOV/platform/issues/710
-Redirect /strategic-plan/ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
+RedirectMatch permanent (?i)^/strategic-plan/$ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: Let's not make this permanent. This URL may be reused to point to future strategic plans.

```suggestion
RedirectMatch (?i)^/strategic-plan/$ https://s3.amazonaws.com/files.consumerfinance.gov/f/strategic-plan.pdf
```

File: cfgov/apache/conf.d/redirects.conf [hunk 4]
```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
-
 # Common misspellings
 RedirectMatch (?i)(.*)morgage(.*) $1mortgage$2
 
 Redirect /assets https://s3.amazonaws.com/files.consumerfinance.gov/a/assets
+
+# Redirects, requested by Rachael Keeler in July 2019
+# https://GHE/CFGOV/platform/issues/3484
+RedirectMatch permanent (?i)^/about-us/advisory-groups/(.*) /about-us/advisory-committees/$1
+Redirect permanent /about-us/blog/understand-your-options-tips-student-loan-borrowers-disabilities /about-us/blog/tips-student-loan-borrowers/
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: I handled this one in Wagtail.

```suggestion
```

File: cfgov/apache/conf.d/redirects.conf [hunk 5]
```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+Redirect permanent /about-us/doing-business-with-us/small-businesses/ /about-us/doing-business-with-us/small-minority-businesses/
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: I handled this one in Wagtail.

```suggestion
```

File: cfgov/apache/conf.d/redirects.conf [hunk 6]
```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+Redirect permanent /about-us/newsroom/student-debt-domino-effect/ /about-us/blog/student-debt-domino-effect/
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: I handled this one in Wagtail.

```suggestion
```

File: cfgov/apache/conf.d/redirects.conf [hunk 7]
```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+Redirect permanent /about-us/project-catalyst/run-a-trial-disclosure/ /about-us/innovation/trial-disclosure-program/
+RedirectMatch permanent (?i)^/askcfpb/search/(.*) /ask-cfpb/search/$1
+Redirect permanent /complaints/ /data-research/consumer-complaints/
+Redirect permanent /consumer-tools/debt-collection/if-creditor-sues-you/ /ask-cfpb/what-should-i-do-if-a-creditor-or-debt-collector-sues-me-en-334/
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: I handled this one in Wagtail.

```suggestion
```

File: cfgov/apache/conf.d/redirects.conf [hunk 8]
```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+Redirect permanent /consumer-tools/everyone-has-a-story/dani-struggling-repay-student-debt/ /consumer-tools/everyone-has-a-story/dani-struggling-to-repay-student-debt/
+Redirect permanent /consumer-tools/money-as-you-grow/book-club/discussion-guides/ /consumer-tools/money-as-you-grow/bookshelf/
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: I handled this one in Wagtail.

```suggestion
```

File: cfgov/apache/conf.d/redirects.conf [hunk 9]
```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+RedirectMatch permanent (?i)^/foia/quarterly/(.*) /foia-requests/foia-electronic-reading-room/cfpb-foia-quarterly-reports/$1
+RedirectMatch permanent (?i)^/find-a-housing-counselor/(.*) /owning-a-home/$1
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: Not sure what happened to get that ` data-original-title=` to appear in the report of a 404 at `https://www.consumerfinance.gov/find-a-housing-counselor/ data-original-title=`, but we definitely don't want to be redirecting it to OAH.

File: cfgov/apache/conf.d/redirects.conf [hunk 10]
```diff
@@ -1017,8 +964,34 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+Redirect permanent /practitioner-resources/students/knowbeforeyouowe/about/ /practitioner-resources/students/knowbeforeyouowe/
+Redirect permanent /practitioner-resources/students/repay/ /paying-for-college/repay-student-debt/
+Redirect permanent /practitioner-resources/students/request-for-information-on-student-loan-servicing/ /paying-for-college/choose-a-student-loan/
+Redirect permanent /pressrelease/treasury-department-announces-senior-leadership-hires-for-the-consumer-financial-protection-bureau/ /about-us/newsroom/treasury-department-announces-senior-leadership-hires-for-the-consumer-financial-protection-bureau/
+Redirect permanent /regulations/final-remittance-rule-amendment-regulation-e/ /policy-compliance/rulemaking/regulations/1005/
+RedirectMatch permanent (?i)^/supervision/(.*) /jobmanager/supervision.html
+RedirectMatch permanent (?)^/wp-content/uploads/(.*) https://s3.amazonaws.com/files.consumerfinance.gov/f/$1
```
Comment (willbarton) [inline cfgov/apache/conf.d/redirects.conf]: This should probably redirect to `https://files.consumerfinance.gov/f/$1` (as should the other s3.amazonaws.com destinations in this file.

The use of s3.amazonaws.com predates our use of CloudFront to provide HTTPS for files.consumerfinance.gov.

File: cfgov/apache/conf.d/redirects.conf [hunk 11]
```diff
@@ -1017,8 +964,34 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+Redirect permanent /policy-compliance/rulemaking/rules-under-development/amendments-equal-credit-opportunity-act-regulation-b-ethnicity-and-race-information-collection/ /policy-compliance/notice-opportunities-comment/archive-closed/amendments-equal-credit-opportunity-act-regulation-b-ethnicity-and-race-information-collection/
+Redirect permanent /practitioner-resources/students/knowbeforeyouowe/about/ /practitioner-resources/students/knowbeforeyouowe/
+Redirect permanent /practitioner-resources/students/repay/ /paying-for-college/repay-student-debt/
+Redirect permanent /practitioner-resources/students/request-for-information-on-student-loan-servicing/ /paying-for-college/choose-a-student-loan/
+Redirect permanent /pressrelease/treasury-department-announces-senior-leadership-hires-for-the-consumer-financial-protection-bureau/ /about-us/newsroom/treasury-department-announces-senior-leadership-hires-for-the-consumer-financial-protection-bureau/
+Redirect permanent /regulations/final-remittance-rule-amendment-regulation-e/ /policy-compliance/rulemaking/regulations/1005/
+RedirectMatch permanent (?i)^/supervision/(.*) /jobmanager/supervision.html
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: There are a few weird things going on with supervision URLs, though none have to do with the job listing page.

I've added the `/supervision/manual/` ➡️ `/policy-compliance/guidance/supervision-examinations/` redirect in Wagtail.

There are other redirects being triggered by [this rule on L558](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/apache/conf.d/redirects.conf#L558) that are problematic, as well – mod_alias's `Redirect` automatically appends anything after the end of the matched path to the end of the destination path, causing 404ing redirects like:

`/guidance/supervision/manual/ecoa-narrative/` ➡️ `/policy-compliance/guidance/supervision-examinations/ecoa-narrative/`

Please adjust L558 as follows:

```diff
-Redirect permanent /guidance/supervision/manual/  /policy-compliance/guidance/supervision-examinations/
+RedirectMatch permanent ^/guidance/supervision/manual/$  /policy-compliance/guidance/supervision-examinations/
```

And the redirect being added on this line should just be deleted.

```suggestion
```

File: cfgov/apache/conf.d/redirects.conf [hunk 12]
```diff
@@ -1017,8 +964,34 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
+RedirectMatch permanent (?)^/wp-content/uploads/(.*) https://files.consumerfinance.gov/f/$1
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: Is this `(?)` doing anything?

```suggestion
RedirectMatch permanent ^/wp-content/uploads/(.*) https://files.consumerfinance.gov/f/$1
```

File: cfgov/apache/conf.d/redirects.conf [hunk 13]
```diff
@@ -1017,8 +964,15 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
...
 
-Redirect /assets https://s3.amazonaws.com/files.consumerfinance.gov/a/assets
+Redirect /assets https://files.consumerfinance.gov/a/assets
+
+# Redirects, requested by Rachael Keeler in July 2019
+# https://GHE/CFGOV/platform/issues/3484
+RedirectMatch permanent (?i)^/about-us/advisory-groups/(.*) /about-us/advisory-committees/$1
+RedirectMatch permanent (?i)^/askcfpb/search/(.*) /ask-cfpb/search/$1
+RedirectMatch permanent (?i)^/how-to-prepare-your-finances-for-buying-a-home/(.*) /owning-a-home/process/prepare/
+Redirect /owning-a-home/help-us-improve/ /owning-a-home/feedback/
```
Comment (Scotchester) [inline cfgov/apache/conf.d/redirects.conf]: Gah! I should have done this one in Wagtail, too. Ah well.

**Pull request merged**